nixos/user.nix

```diff
@@ -0,0 +1,5 @@
+{ 
+  name = "carlos"; 
+  hostname = "solrac";
+  luks-device-uuid = "/dev/disk/by-uuid/b9d4779c-0148-4c33-9048-842f32efca8a";
```

Comment:
`/dev/disk/by-uuid/` appears to be duplicated here and where the variable is used. 

The variable should `disk-uuid` luks just refers to an encryption technique that we are currently using